extensions/extensions-helm/aissemble-configuration-store-chart/templates/webhook-init-job.yaml

```diff
@@ -0,0 +1,45 @@
+{{ if .Values.webhook.enable }}
+apiVersion: batch/v1
+kind: Job
```

Comment:
Q: These are done as jobs instead of init-containers so that the main deployment can be less privileged?

Reply:
Copied the approach from spark-operator. I supposed you could do them as init containers on the main deployment too, however, this does have the added benefit of using a separate service account to control the privileges.